### ISSUE TITLE
Cancel not referenced

### ISSUE THREAD

**Issue by srugano** [Dirk-Sandberg/FarmersMarketFinderTutorial#2]
This line produces an error: `AttributeError: 'NoneType' object has no attribute 'cancel'`

https://github.com/Dirk-Sandberg/FarmersMarketFinderTutorial/blob/30295797691bf6a88bfebee077a4f14b5785dcf2/part2/farmersmapview.py#L12

**Comment by Dirk-Sandberg** [comment]
Try changing the try/except block to something like:

`
if self.getting_markets_timer:
    self.getting_markets_timer.cancel()
    self.getting_markets_timer = None
`
Let me know if that addresses the problem!

**Comment by srugano** [comment]
Did that and everything worked. Thanks. But you should add a `pip freeze > requirements.txt` file to outline the version of the packages you used. For example, Kivy 2.0.0 removed some of the classes you have used extensively. It should be Kivy==1.1.1, KivyMD==0.104.0 and Mapview==1.0.4. 

Thanks

**Comment by Dirk-Sandberg** [comment]
You're totally right -- if you have that already set up would you mind opening a PR?
